NitroxClient/Debuggers/SceneDebugger.cs

```diff
@@ -237,31 +237,43 @@ private void RenderTabScenes()
             using (new GUILayout.VerticalScope("Box"))
             {
                 GUILayout.Label("All scenes", "header");
-                for (int i = 0; i < SceneManager.sceneCountInBuildSettings; i++)
+                for (int i = 0; i < SceneManager.sceneCountInBuildSettings+1; i++)
                 {
-                    Scene currentScene = SceneManager.GetSceneByBuildIndex(i);
-                    string path = SceneUtility.GetScenePathByBuildIndex(i);
+                    Scene currentScene;
+                    string path = "";
+                    if (i == SceneManager.sceneCountInBuildSettings)
+                    {
+                        currentScene = NitroxBootstrapper.Instance.gameObject.scene; // bit of a hack
+                    }
+                    else
+                    {
+                        currentScene = SceneManager.GetSceneByBuildIndex(i);
+                        path = SceneUtility.GetScenePathByBuildIndex(i);
+                    }
+
+                    Log.Info($"Current Scene: {currentScene.name} current Index {i}");
+
                     bool isSelected = selectedScene.IsValid() && currentScene == selectedScene;
                     bool isLoaded = currentScene.isLoaded;
 
                     using (new GUILayout.HorizontalScope("Box"))
                     {
-                        if (GUILayout.Button($"{(isSelected ? ">> " : "")}{i}: {path.TruncateLeft(35)}", isLoaded ? "sceneLoaded" : "label"))
+                        if (GUILayout.Button($"{(isSelected ? ">> " : "")}{i}: {(currentScene.name == "DontDestroyOnLoad" ? currentScene.name : path.TruncateLeft(35))}", isLoaded ? "sceneLoaded" : "label"))
```

Comment:
Recommend storing the `currentScene.name == ` condition as a bool and reuse that for performance/readability reasons.